roles/common/tasks/password-policy.yml

```diff
@@ -2,10 +2,12 @@
 - name: remove invalid configuration for pam_unix
   lineinfile: dest=/etc/pam.d/common-password regexp=(pam_unix.so) state=absent
               line="password [success=7 default=ignore] pam_unix.so obscure sha512"
+  when: os == 'ubuntu'
```

Comment:
You may want to consider combining all OS tasks into a block with a single when condition. Group all Ubuntu together and all RHEL together. You can also add tags to the block if they apply to all items within.